src/styles/base.scss

```
@@ -1,10 +1,10 @@
 @use 'reset-css/sass/reset.scss';
 
+.root {
+  font-size: 1.6rem;
+}
+
 html,
 body {
   font-size: 62.5%;
 }
```

Trocar isso de posição e ajustar as respectivas propriedades nos lugares certos 
```suggestion
:root {
  font-size: 62.5%;
}

body {
  font-size: 1.6rem;
}
```

---

Esse `:root` pode ser alterado por um `html`, vai do teu gosto